### PR TITLE
RHDEVDOCS 5886 clarify uninstallation for Pipelines

### PR DESCRIPTION
**Pull request by mramendi**
Version(s):

please cp to pipelines-docs-1.11, pipelines-docs-1.12, pipelines-docs-1.13, pipelines-docs-1.14, pipelines-docs-1.15

Issue:

RHDEVDOCS 5886

Link to docs preview:

https://71402--ocpdocs-pr.netlify.app/openshift-pipelines/latest/install_config/uninstalling-pipelines.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by piyush-garg** [comment]
i dont think we need to replace it with CRD

**Review by concaf — APPROVED** [review]

**Comment by concaf** [comment]
though i agree with @piyush-garg that it should be CRs and not CRDs

File: modules/op-deleting-the-pipelines-component-and-custom-resources.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@
 [id='op-deleting-the-pipelines-component-and-custom-resources_{context}']
 = Deleting the {pipelines-title} components and Custom Resources
 
-Delete the Custom Resources (CRs) created by default during installation of the {pipelines-title} Operator.
```
Comment (piyush-garg) [inline modules/op-deleting-the-pipelines-component-and-custom-resources.adoc:8]: CRs is correct, not CRD

File: modules/op-deleting-the-pipelines-component-and-custom-resources.adoc [hunk 2]
```diff
@@ -20,12 +20,12 @@ Delete the Custom Resources (CRs) created by default during installation of the
 +
 [NOTE]
 ====
-Deleting the CRs will delete the {pipelines-title} components, and all the tasks and pipelines on the cluster will be lost.
```
Comment (piyush-garg) [inline modules/op-deleting-the-pipelines-component-and-custom-resources.adoc:23]: CRs is correct, not CRD

File: modules/op-deleting-the-pipelines-component-and-custom-resources.adoc [hunk 3]
```diff
@@ -20,12 +20,12 @@ Delete the Custom Resources (CRs) created by default during installation of the
...
+Deleting the CRDs will delete the {pipelines-title} components, and all the tasks and pipelines on the cluster will be lost.
 ====
 
-. Click *Delete* to confirm the deletion of the CRs.
+. Click *Delete* to confirm the deletion of the CRDs.
 
 [IMPORTANT]
 ====
-Repeat the procedure to find and remove CRs of optional components such as {tekton-hub} before uninstalling the Operator. If you uninstall the Operator without removing the CRs of optional components, you cannot remove them later.
+Repeat the procedure to find and remove all remaining CRDs which contains `tekton.dev` in their name before uninstalling the Operator. If you uninstall the Operator without removing these CRDs, you cannot remove them later.
```
Comment (piyush-garg) [inline modules/op-deleting-the-pipelines-component-and-custom-resources.adoc:30]: make CRs in place of CRD

**Comment by aireilly** [comment]
This branch has been repointed to target pipelines-docs-main. pipelines-docs is no longer in use.

File: install_config/uninstalling-pipelines.adoc [hunk 4]
```diff
@@ -8,21 +8,24 @@ toc::[]
 
 Cluster administrators can uninstall the {pipelines-title} Operator by performing the following steps:
 
-. Delete the Custom Resources (CRs) that were added by default when you installed the {pipelines-title} Operator.
-. Delete the CRs of the optional components such as {tekton-hub} that depend on the Operator.
+. Delete the `TektonConfig` Custom Resource (CR), as well as the CRs for the optional components, `TektonHub` and `TektonChains`, if these CRs exist.
```
Comment (ppitonak) [inline install_config/uninstalling-pipelines.adoc:11]: optional CRs first, TektonConfig last

File: install_config/uninstalling-pipelines.adoc [hunk 5]
```diff
@@ -8,21 +8,24 @@ toc::[]
...
 +
 [CAUTION]
 ====
-If you uninstall the Operator without removing the CRs of optional components, you cannot remove them later.
+If you uninstall the Operator without removing the CRs of optional components, you cannot remove the components later.
 ====
 . Uninstall the {pipelines-title} Operator.
+. Delete the Custom Resource Definitions (CRDs) of the `operator.tekton.dev` group.
```
Comment (ppitonak) [inline install_config/uninstalling-pipelines.adoc:18]: While it's necessary to remove optional components' CRs, do we also need to remove CRDs? 😕

**Review by ppitonak — CHANGES_REQUESTED** [review]

File: install_config/uninstalling-pipelines.adoc (hunk 5 above)
Comment (mramendi) [reply install_config/uninstalling-pipelines.adoc:18]: @ppitonak this is what @piyush-garg told me on Slack. @piyush-garg can you confirm?

**Review by ppitonak — APPROVED** [review]

File: install_config/uninstalling-pipelines.adoc [hunk 6]
```diff
@@ -8,21 +8,24 @@ toc::[]
 
 Cluster administrators can uninstall the {pipelines-title} Operator by performing the following steps:
 
-. Delete the Custom Resources (CRs) that were added by default when you installed the {pipelines-title} Operator.
-. Delete the CRs of the optional components such as {tekton-hub} that depend on the Operator.
+. Delete the Custom Resources (CRs) for the optional components, `TektonHub` and `TektonChains`, if these CRs exist, and then delete the `TektonConfig` CR.
```
Comment (piyush-garg) [inline install_config/uninstalling-pipelines.adoc:11]: TektonChains is not optional anymore from 1.12 i think

Comment (mramendi) [reply install_config/uninstalling-pipelines.adoc:11]: @piyush-garg I have replaced "optional" with "additional", is everything good now?

Comment (piyush-garg) [reply install_config/uninstalling-pipelines.adoc]: so this step is valid only for optional component such as `TektonHub` and `TektonResults` and in future `Manual Approval`. This is not valid for `TektonChain` as that is not optional component and is part of `TektonConfig`

**Comment by piyush-garg** [comment]
lgtm

**Review by kcarmichael08 — COMMENTED** [review]
Some minor issues and questions, but looks good overall!

**Comment by mramendi** [comment]
/remove-label merge-review-needed

typo found

**Pull request merged**